app/forms/state_file/notification_preferences_form.rb

```diff
@@ -4,16 +4,23 @@ class NotificationPreferencesForm < QuestionsForm
     before_validation :normalize_phone_numbers
 
     validates :phone_number, e164_phone: true, if: -> { attributes_for(:intake)[:sms_notification_opt_in] == "yes" }
-    validates :email_address, 'valid_email_2/email': true, if: -> { attributes_for(:intake)[:email_notification_opt_in] == "yes" }
-    validates :email_address, presence: true, if: -> { attributes_for(:intake)[:email_notification_opt_in] == "yes" }
+    validates :email_address, 'valid_email_2/email': true, if: -> { should_validate_email? }
+    validates :email_address, presence: true, if: -> { should_validate_email? }
     validate :at_least_one_selected
 
     def save
-      @intake.update(attributes_for(:intake))
+      email_attributes = should_validate_email? ? {} : {email_address: @intake.email_address}
+
+      @intake.update(attributes_for(:intake).merge(email_attributes))
```

Comment:
[q] do we need to update the intake with the existing `email_address` if it's already on the intake?

Reply:
yeah it updates to nil otherwise

Reply:
oh right, i guess the email param still comes through even for the statement-only version